ui/src/app/machines/views/MachineDetails/MachineStorage/AvailableStorageTable/AddPartition/AddPartition.tsx

```diff
@@ -70,14 +70,13 @@ export const AddPartition = ({
     }
   }, [closeExpanded, saved]);
 
-  if (machine && "disks" in machine && "supported_filesystems" in machine) {
+  if (machine && "supported_filesystems" in machine) {
```

Comment:
It's basically there to tell TS that the `machine` returned from `machineSelectors.getById()` is of type `MachineDetails` instead of `Machine`. We're using it in a few other places in the new machine details components but we should probably just create a util like:
``` typescript
export const isMachineDetails(machine: Machine | MachineDetails): machine is MachineDetails => {
  // something that 100% determines something is MachineDetails
};
```

Reply:
Created an issue https://github.com/canonical-web-and-design/maas-ui/issues/1949